api/controllers/PostController.js

```diff
@@ -97,6 +97,20 @@ const queryForTag = function (req, res) {
   }))
 }
 
+var queryForTagInAllCommunities = function (req, res) {
+  if (TokenAuth.isAuthenticated(res)) {
+    if (!RequestValidation.requireTimeRange(req, res)) return
```

Comment:
i think you want to drop these two if-clauses entirely? we don't need to support this janky token auth anymore and returning null here will cause an error for upstream code that expects a promise anyway.